### DISCUSSION TITLE
Code of Conduct document

### DISCUSSION THREAD

**Discussion by miertla** [beetbox/beets#4782] [General]
Hi,

I love the project and find it interesting as I LOVE music! I was reading over the documentation as I was setting beets up on my own device (Mac) and noticed that there is no Code of Conduct document. I might have missed it on here or the Discourse forum, but I believe this is an important document that seems to be missing. I know this creates additional work for the maintainers, but would recommend adapting the Contributor Covenant Code of Conduct version 1.4: https://www.contributor-covenant.org/version/1/4/code-of-conduct/ if the maintainers agree that this is something that is important to this project (or another version if that fits their needs better). Thanks and I apologize if I missed it somewhere.

**Comment by JOJ0** [comment]
I think we had another issue/discussion topic already where code of conduct was suggested and it sadly got ignored. I'm not sure yet why this is important and I will try to do my homework learning it...

**Reply by JOJ0** [reply]
Hi @miertla the PR that tries to add a CoC doc is still open. Have a look: https://github.com/beetbox/beets/pull/4531

**Reply by miertla** [reply]
Thank you! I missed this when I was searching for an already opened issue regarding this.
